src/position.cpp

```diff
@@ -1546,10 +1559,28 @@ Value Position::see(Move m) const {
 #endif
 
       // Add the new entry to the swap list
+#ifdef ANTI
+      if (is_anti())
+          swapList[slIndex] = -swapList[slIndex - 1] + PieceValueAnti[MG][captured];
+      else
+#endif
       swapList[slIndex] = -swapList[slIndex - 1] + PieceValue[MG][captured];
 
       // Locate and remove the next least valuable attacker
       captured = min_attacker<PAWN>(byTypeBB, to, stmAttackers, occupied, attackers);
+#ifdef ANTI
```

Comment:
Sounds good to me. With a separate `min_attacker_anti` we could also consider using the reverse order (maybe except for the king), e.g. take with the queen first, then rook, bishop, knight, pawn, and king to get a more accurate SEE.